src/css/header.css

```
@@ -19,9 +19,337 @@ html.is-clipped--navbar {
+  min-width: 200px;
+  max-width: 320px;
+  background: #fff;
+  border: 1px solid var(--grey-200-new, #c3c4c6);
+  border-radius: 8px;
+  box-shadow: 0 4px 16px rgba(0, 0, 0, 0.12);
+  padding: 6px;
+  z-index: 100;
+  animation: bcDropdownIn 0.15s ease-out;
+}
+
+@keyframes bcDropdownIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "bcDropdownIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)